app/models/admin.js

```diff
@@ -0,0 +1,8 @@
+import DS from 'ember-data';
+
+export default DS.Model.extend({
+  name: DS.attr('string'),
+  description: DS.attr('string'),
+  category: DS.attr('string'),
+  price: DS.attr('string')
+});
```

Comment:
Why is this model called `admin`? You are not modifying `admin` users, you are changing `menu-item`s